JSDemos/Demos/List/ListSelection/description.md

```diff
@@ -1 +1,11 @@
-The **List** UI component offers three item selection modes: _&laquo;all&raquo;_, _&laquo;single&raquo;_ and _&laquo;multi&raquo;_. Note that when the current mode is _&laquo;all&raquo;_, the **List** allows a&nbsp;user to&nbsp;select all items at&nbsp;once by&nbsp;clicking the _Select All_ check box. The **selectAllMode** property allows you to&nbsp;decide whether all items should be&nbsp;selected across all pages or&nbsp;within the currently rendered pages only.
\ No newline at end of file
+The **List** offers four selection modes: *none*, *all*, *single*, and *multi*. 
+
+Selection is disabled by default. To enable it, pass one of the mentioned above values to the [selectionMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode) property.
+
+Users can click or tap on the **List** item to select it. Selected items become shaded. It is possible to add selection checkboxes or radio buttons to list items. To do so, set the [showSelectionControls](/Documentation/ApiReference/UI_Components/dxList/Configuration/#showSelectionControls) property to **true**. This property also must be set to *true* when you use the *all* selection mode.
```

Comment:
Since in this demo showSelectionControls is enabled, we should start the description from the situation when the controls are already visible.

```suggestion
List items can display checkboxes or radio buttons that help users select the items. To show these controls, enable the [showSelectionControls](/Documentation/ApiReference/UI_Components/dxList/Configuration/#showSelectionControls) property. Otherwise, users click or tap list items to select them. The **showSelectionControls** property must also be enabled when you use the *"all"* **selectionMode** so that the Select All checkbox is visible.
```